xml/obs_admin_appliance.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="cha.obs.admin.appliance">
+ <title>How to use the OBS appliances in production</title>
+ <para>
+   Generic overview how to use the OBS appliances in production. The appliances can be just
```

Comment:
The introduction sounds a bit bumpy to me. It could be smother starting with "This is" or "This provides a" or by replacing the whole sentence